ui/css/perf.css

```diff
@@ -450,27 +450,27 @@ li.pagination-active.active > button {
 
 /* graph colors */
 .orange {
-  border-left-color: #ffba4a;
+  border-left-color: #ffb851;
 }
 
 .firered {
-  border-left-color: #c92b2a;
+  border-left-color: #c92d2f;
```

Comment:
For CSS names we use snake case (multiple words are separated with a hyphen so its easier to read).